src/components/ToolsPageContent.tsx

```diff
@@ -188,6 +188,18 @@ const docsTools = [
     category: "",
     featured: false,
   },
+  {
+    id: 40,
```

Comment:
This page is for documentation tools. You do not need to add it here. Please revert the changes in this file.